packages/dataclass-editor/src/master/DataClassMasterContent.tsx

```diff
@@ -131,7 +180,14 @@ export const DataClassMasterContent = () => {
           <TableResizableHeader headerGroups={table.getHeaderGroups()} onClick={() => selectRow(table)} />
           <TableBody>
             {table.getRowModel().rows.map(row => (
-              <ValidationRow key={row.id} row={row} isReorderable={table.getState().sorting.length === 0} />
+              <ValidationRow
+                key={row.id}
+                row={row}
+                isReorderable={table.getState().sorting.length === 0}
+                onDrag={!row.getIsSelected() ? () => table.resetRowSelection() : undefined}
+                onClick={event => handleMultiSelectOnCtrlRowClick(table, row, event)}
```

Comment:
I'd also like to see the possibility of holding `SHIFT` while clicking to select everything from the previously clicked row to the newly clicked row. But that's something for another day.